app/assets/javascripts/application.js.erb

```diff
@@ -597,19 +534,73 @@ onPageLoad(() => {
   });
 
   on('click', 'button.story-preview', (event) => {
-    Lobsters.previewStory(parentSelector(event.target, 'form'));
+    Lobster.previewStory(parentSelector(event.target, 'form'));
   });
 
   // Comment Related Functions
 
+  on('click', 'a.comment-disowner', (event) => {
+    if (confirm("Are you sure you want to disown this comment?")) {
+      let li = parentSelector(event.target, '.comment');
+      fetchWithCSRF('/comments/' + li.getAttribute('data-shortid') + '/disown', {
+        method: 'post'
+      })
+        .then(response => response.text())
+        .then(text => replace(li, text));
+    }
+  });
+
+  on("click", "a.comment_replier", (event) => {
+    if (!Lobsters.curUser) {
+      Lobster.bounceToLogin();
+    }
+
+    const comment = parentSelector(event.target, '.comment');
+    const commentId = comment.getAttribute('id');
+
+    // Check to make sure we don't already have a reply box.
+    if (document.getElementById('reply_form_' + commentId)) {
+      return false;
+    }
+
+    // Inserts "> " on quoted text.
+    let sel = document.getSelection().toString() || '';
+    if (sel != "") {
+      sel = sel.split("\n").map(s => "> " + s + '\n').join('');
+      sel += "\n";
+    }
+
+    let div = document.createElement('div');
+    if (document.querySelector('.comments_subtree')) {
+      parentSelector(comment, '.comments_subtree').lastElementChild.prepend(div);
+    } else {
+      console.log('lastchild', comment.parentElement.lastElementChild.prepend(div))
+      console.log('not last child', comment.parentElement.prepend(div))
+      comment.parentElement.lastElementChild.prepend(div);
+    }
+
+    fetch('/comments/' + comment.getAttribute('data-shortid') + '/reply')
+      .then(response => {
+        response.text().then(text => {
+          div.innerHTML = text;
+          div.setAttribute('id', 'reply_form_' + commentId );
+          var ta = div.querySelector("textarea");
+          ta.textContent = sel;
+          ta.setSelectionRange(sel.length, sel.length);
```

Comment:
This puts the cursor at the end? Nice addition, please add a comment to confirm if so.